contracts/suilend/sources/obligation.move

```diff
@@ -1221,4 +1370,57 @@ module suilend::obligation {
             user_reward_manager_index
         }
     }
-}
+
+    fun get_ltvs<P>(
```

Comment:
nit: this function is a bit hard to read because of all the branches. in general, I prefer to return early in a function to avoid nesting if-elses, if that makes sense.

Reply:
Makes sense. Refactored